File: Easy/LCE - 1160.js
/**
 * @param {string[]} words
 * @param {string} chars
 * @return {number}
 */
var countCharacters = function(words, chars) {
    
    let sum = 0
    const arr = Array.from(chars).sort()
    
    words.forEach(word => {
        let temp = Array.from(word).sort()
        let j = 0;
        for(let i of arr) {
            if(i===temp[j])
                j++
        }
        if(j==temp.length)
            sum += temp.length
    })
    
    return sum
    
};

File: Easy/Python - Solutions/LCE_1160.py
class Solution:
    def countCharacters(self, words: List[str], chars: str) -> int:
        ans = 0
        for val in words:
            flag = 1
            for i in set(val):
                if chars.count(i) < val.count(i):
                    flag = 0
                    break
            if flag == 1:
                ans += len(val)

        return ans
